frontend/packages/topology/src/behavior/useDndDrag.tsx

```diff
@@ -171,29 +176,34 @@ export const useDndDrag = <
                       if (op && operationChangeEvents) {
                         dndManager.beginDrag(idRef.current, op, ...operationChangeEvents.begin);
                         dndManager.drag(...operationChangeEvents.drag);
-                        operationChangeEvents = null;
+                        operationChangeEvents = undefined;
                       }
                     }
                   }
-                });
+                };
                 d3.select(node.ownerDocument)
                   .on(
                     'keydown.useDndDrag',
-                    actionAsync(async () => {
+                    flow(function*() {
                       const e = d3.event as KeyboardEvent;
                       if (e.key === 'Escape') {
                         if (dndManager.isDragging() && dndManager.cancel()) {
-                          operationChangeEvents = null;
+                          operationChangeEvents = undefined;
```

Comment:
Why are you favouring `undefined` over `null`? I can't think of a reason why you'd want to swap out `null`s for `undefined`s shy of maybe a typeof check for object - which you're not doing.

Reply:
See comment above: https://github.com/openshift/console/pull/3484#discussion_r353378631